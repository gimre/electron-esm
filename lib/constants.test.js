
const { MODULE_DIR, ... constants } = require( './constants' )

test( 'are set correctly', ( ) => {
    expect( constants ).toMatchSnapshot( )
} )