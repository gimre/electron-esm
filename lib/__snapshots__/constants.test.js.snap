// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`are set correctly 1`] = `
Object {
  "DEFAULT_OPTIONS": Object {
    "cjsProtocol": "cjs",
    "esmProtocol": "esm",
    "root": "./",
  },
  "ERR_APP_ALREADY_READY": "you can only use electron-esm before app.ready",
  "ERR_FILE_NOT_FOUND": -6,
  "ERR_NOT_MAIN_PROCESS": "you can only use electron-esm in the main process",
  "MIME_TYPE": "text/javascript",
  "PROCESS_TYPE_MAIN": "browser",
}
`;
